ecr-login/api/client.go

```diff
@@ -85,10 +91,13 @@ func (self *defaultClient) GetCredentialsByRegistryID(registryID string) (*Auth,
 	cachedEntry := self.credentialCache.Get(registryID)
 	if cachedEntry != nil {
 		if cachedEntry.IsValid(time.Now()) {
-			log.Debugf("Using cached token for %s", registryID)
+			logrus.WithField("registry", registryID).Debug("Using cached token for")
```

Comment:
nit: probably the message would read fine just as "Using cached token"

Reply:
Fixed